help_text/4c1560dca6a927e575ca4fc7f77516ad.md

```diff
@@ -0,0 +1,13 @@
+This error means that there are unencoded ampersands (&) in your markup.  HTML uses the ampersand character to start a character reference, and is thus a reserved symbol.  Instead of typing `&`, type the character reference `&amp;` instead.
+
+The following example is **not valid** because it does not use a character reference:
+
+```
+today & tomorrow
```

Comment:
fixed in https://github.com/mfairchild365/metric_w3c_html/commit/aeedcc35ee0bbac79d8a58a98bd7fc517cb5050d